interface/src/app/utils.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Unit tests.
+ */
+
+// Test the hyperGeometricTest function
+describe('Test for hypergeometric test function in utils.js', function() {
+  beforeEach(module('adage.utils'));
+
+  var mockService;
+
+  beforeEach(function() {
+    angular.mock.inject(function($injector) {
```

Comment:
Ah gotcha - totally, I have found them pretty tricky to understand too. I will change this, thanks!